src/main/resources/edu/cornell/kfs/gl/cu-spring-gl.xml

```diff
@@ -338,5 +338,22 @@
 		<property name="persistenceStructureService">
 			<ref bean="persistenceStructureService" />
 		</property>
-	</bean> 	 		     
+	</bean> 	 
+	
+	<bean id="glPostExpenditureTransaction" parent="glPostExpenditureTransaction-parentBean" />
+	<bean id="glPostExpenditureTransaction-parentBean" class="edu.cornell.kfs.gl.batch.service.impl.CuPostExpenditureTransaction" abstract="true">
```

Comment:
You don't need to override the parent bean in this case. You just need the "glPostExpenditureTransaction" override and then set the "class" on it.